### ISSUE TITLE
shortcuts cannot be opened

### ISSUE THREAD

**Issue by anschuetz** [gonzula/splash#20]
when I press the "execute" button at the upper right of the screen, shortcuts-app is being started and says: Shortcut cannot be opened. Import of shortcuts via files is not supported. Share shortcuts via icloud.  (Translation may be weak... I've got the german iOS)

so I cannot test nor use the shortcuts.

**Comment by pfgithub** [comment]
This is because ios 13 removed the ability to import shortcuts from a file. There is a workaround where you add the .shortcut to icloud drive and get link, but you'll have to use splash on a computer to get the .shortcut file.

**Comment by anschuetz** [comment]
thanks! I'll try that :)

**Comment by gonzula** [comment]
Yeah, that’s right. iOS 13 have made things pretty difficult for Splash. Mainly because now I can’t export the `.shortcut` files to “reverse engineer” them.

**Comment by pfgithub** [comment]
You can export .shortcut files by
```
Get My Shortcuts
Choose From Menu
Set Name shortcut.plist (you might have to check the checkbox)
Show Result
```
You should also be able to just share > share file normally but I haven't checked if you can do that or not

**Comment by pfgithub** [comment]
You can export .shortcut files by
```
Get My Shortcuts
Choose From Menu
Set Name shortcut.plist (you might have to check the checkbox)
Show Result
```
You should also be able to just share > share file normally but I haven't checked if you can do that or not

**Comment by gonzula** [comment]
Wow, I wasn't aware of that way of exporting.
And `share > share file` doesn't work anymore. It was the the way I used to do.
